internal/sql/migrations.go

```diff
@@ -217,6 +220,158 @@ var (
 					return performMigration(ctx, tx, migrationsFs, dbIdentifier, "00017_unix_ms", log)
 				},
 			},
+			{
+				ID: "00018_settings",
+				Migrate: func(tx Tx) error {
+					log.Infof("performing %s migration '00018_settings'", dbIdentifier)
```

Comment:
Is there a way we can get rid of the Go part of the migration? Theoretically, the settings should all exist and be valid. We can also add or remove fields using the json operators for SQLite and MySQL.

Reply:
I really did give it the good ol' college try but I gave up too quickly I think. I don't like if the SQLite differs from the MySQL migration, especially if the migration is easy. It's just more to manage (and review) and it's an extra risk in a way, SQLite might be fine and a MySQL bug could go by unnoticed. I didn't like it either though, I'm happy you pushed back on it.